admin_guide/configuring_gce.adoc

```diff
@@ -0,0 +1,67 @@
+= Configuring for GCE
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+OpenShift can be configured to access an link:https://cloud.google.com/compute/docs/disks/[GCE infrastructure], 
+including link:persistent_storage/persistent_storage_gce.html[using GCE volumes as persistent storage] for application data.
+After GCE is configured properly, some additional configurations will need to be completed on the OpenShift hosts.
+
+
+== Configuring GCE Variables
+To set the required GCE variables, create a *_/etc/aws/aws.conf_* file with
+the following contents on all of your OpenShift hosts, both masters and nodes:
+
+
+== Configuring Masters
+
+Edit or
+link:master_node_configuration.html#creating-new-configuration-files[create] the
+master configuration file on all masters
+(*_/etc/openshift/master/master-config.yaml_* by default) and update the
+contents of the `*apiServerArguments*` and `*controllerArguments*` sections:
+
+====
+[source,yaml]
+----
+kubernetesMasterConfig:
+  ...
+  apiServerArguments:
+    cloud-provider:
+      - "gce"
+  controllerArguments:
+    cloud-provider:
+      - "gce"
+
+----
+====
+
+== Configuring Nodes
+
+Edit or
+link:master_node_configuration.html#creating-new-configuration-files[create] the
+node configuration file on all nodes
+(*_/etc/openshift/node-<hostname>/node-config.yaml_* by default) and update the
+contents of the `*kubeletArguments*` section:
+
+====
+[source,yaml]
+----
+kubeletArguments:
+  cloud-provider:
+    - "gce"
+
+----
+====
+
+
+
+(Re-)start OpenShift services on the master and all nodes
```

Comment:
Was this cut off or is this indeed all?

s/(Re-)start/Start or restart/

Missing period.